app/javascript/components/public_health/PatientsTable.js

```diff
@@ -565,7 +565,7 @@ class PatientsTable extends React.Component {
         <Nav variant="tabs" activeKey={this.state.query.tab}>
           {Object.entries(this.props.tabs).map(([tab, tabProps]) => {
             return (
-              <Nav.Item key={tab} className={tab === 'all' ? 'ml-auto' : ''}>
+              <Nav.Item key={tab} className={tab === 'all' ? 'ml-xl-auto' : ''}>
```

Comment:
so I have the styling swap happening at the xl breakpoint because that's the closest to when all the tabs are together.  I think it still makes sense that on bigger screens, the all monitorees stays on the right.  Let me see if I can incorporate this into the media css instead

Reply:
I wrapped this into the media css instead and it's a lot smoother now: https://github.com/SaraAlert/SaraAlert/pull/931/commits/58106feefac1e7e3afcff6d1721c18f8f1e27684